templates/docs/examples/layouts/documentation.html

```diff
@@ -4,7 +4,7 @@
 {% block content %}
 {% include "docs/examples/layouts/_global-nav.html" %}
 
-<header id="navigation" class="p-navigation is-dark">
+<header id="navigation" class="p-navigation is-dark" role="banner">
```

Comment:
Docs say that `header` is equivalent of `banner`: "By default, the HTML5 <header> element has an identical meaning to the banner landmark, unless it is a descendant of <aside>, <article>, <main>, <nav>, or <section>.".

So maybe we can use that instead?

Reply:
Oh, it's a main navigation not side navigation... I got confused.

So it safe to drop banner role. It is a header already, and it will be top level element (not in any section, etc).